src/parser/ui/TalentAggregateBar.tsx

```diff
@@ -13,23 +13,25 @@ type Props = {
   scaleFactor?: number;
   subSpecs?: TalentAggregateBarSpec[];
   subSpecPercents?: number[];
+  wide: boolean;
 };
 
 const TalentAggregateBar = ({
   amount,
   percentTotal,
   barColor,
   barTooltip,
-  scaleFactor,
+  scaleFactor = 1,
   subSpecs,
   subSpecPercents,
+  wide = true,
```

Comment:
Should change the default to false and that way you only need to add `.. wide/>` for the bar to be wide

Reply:
Done